<launch>
    <node pkg='segmentation_tools' type='segment_bounding_box' name='segment_bounding_box' output='screen'>
        <!--param name='min_x' value='-1'/>
        <param name='min_y' value='-1'/>
        <param name='min_z' value='-1'/>
        <param name='max_x' value='1'/>
        <param name='max_y' value='1'/>
        <param name='max_z' value='1'/-->
        <remap from='segmented_cloud' to='bounded_cloud' />
    </node>
    <node pkg='segmentation_tools' type='segment_ransac' name='segment_ransac' output='screen'>
        <remap from='/camera/depth/points' to='bounded_cloud' />
    </node>
</launch>
